jobs/collect_logs.sh

```diff
@@ -6,53 +6,43 @@ source /usr/local/src/cinder-ci/jobs/utils.sh
-run_wsmancmd_with_retry 3 $hyperv01 $WIN_USER $WIN_PASS 'powershell -executionpolicy remotesigned get-netfirewallprofile ^| Select-Object * >> \\'$DEVSTACK_FLOATING_IP'\openstack\logs\'${hyperv01%%[.]*}'\firewall.log'
-run_wsmancmd_with_retry 3 $hyperv01 $WIN_USER $WIN_PASS 'powershell -executionpolicy remotesigned get-process ^| Select-Object * >> \\'$DEVSTACK_FLOATING_IP'\openstack\logs\'${hyperv01%%[.]*}'\get_process.log'
-run_wsmancmd_with_retry 3 $hyperv01 $WIN_USER $WIN_PASS 'powershell -executionpolicy remotesigned get-service ^| Select-Object * >> \\'$DEVSTACK_FLOATING_IP'\openstack\logs\'${hyperv01%%[.]*}'\get_service.log'
+run_wsmancmd_with_retry 3 $hyperv01 $WIN_USER $WIN_PASS 'powershell -executionpolicy remotesigned get-netadapter ^| Select-object * >> \\'$DEVSTACK_FLOATING_IP'\openstack\logs\hyperv01\get_netadapter.log'
+run_wsmancmd_with_retry 3 $hyperv01 $WIN_USER $WIN_PASS 'powershell -executionpolicy remotesigned get-vmswitch ^| Select-object * >> \\'$DEVSTACK_FLOATING_IP'\openstack\logs\hyperv01\get_vmswitch.log'
+run_wsmancmd_with_retry 3 $hyperv01 $WIN_USER $WIN_PASS 'powershell -executionpolicy remotesigned get-WmiObject win32_logicaldisk ^| Select-object * >> \\'$DEVSTACK_FLOATING_IP'\openstack\logs\hyperv01\disk_free.log'
+run_wsmancmd_with_retry 3 $hyperv01 $WIN_USER $WIN_PASS 'powershell -executionpolicy remotesigned get-netfirewallprofile ^| Select-Object * >> \\'$DEVSTACK_FLOATING_IP'\openstack\logs\hyperv01\firewall.log'
+run_wsmancmd_with_retry 3 $hyperv01 $WIN_USER $WIN_PASS 'powershell -executionpolicy remotesigned get-process ^| Select-Object * >> \\'$DEVSTACK_FLOATING_IP'\openstack\logs\hyperv01\get_process.log'
+run_wsmancmd_with_retry 3 $hyperv01 $WIN_USER $WIN_PASS 'powershell -executionpolicy remotesigned get-service ^| Select-Object * >> \\'$DEVSTACK_FLOATING_IP'\openstack\logs\hyperv01\get_service.log'
 
-run_wsmancmd_with_retry 3 $hyperv01 $WIN_USER $WIN_PASS 'sc qc nova-compute >> \\'$DEVSTACK_FLOATING_IP'\openstack\logs\'${hyperv01%%[.]*}'\nova_compute_service.log'
-run_wsmancmd_with_retry 3 $hyperv01 $WIN_USER $WIN_PASS 'sc qc neutron-hyperv-agent >> \\'$DEVSTACK_FLOATING_IP'\openstack\logs\'${hyperv01%%[.]*}'\neutron_hyperv_agent_service.log'
+run_wsmancmd_with_retry 3 $hyperv01 $WIN_USER $WIN_PASS 'sc qc nova-compute >> \\'$DEVSTACK_FLOATING_IP'\openstack\logs\hyperv01\nova_compute_service.log'
+run_wsmancmd_with_retry 3 $hyperv01 $WIN_USER $WIN_PASS 'sc qc neutron-hyperv-agent >> \\'$DEVSTACK_FLOATING_IP'\openstack\logs\hyperv01\neutron_hyperv_agent_service.log'
 
-run_wsmancmd_with_retry 3 $hyperv02 $WIN_USER $WIN_PASS 'powershell -ExecutionPolicy RemoteSigned Copy-Item -Recurse C:\OpenStack\Log\* \\'$DEVSTACK_FLOATING_IP'\openstack\logs\'${hyperv02%%[.]*}'\'
+run_wsmancmd_with_retry 3 $hyperv02 $WIN_USER $WIN_PASS 'powershell -ExecutionPolicy RemoteSigned Copy-Item -Recurse C:\OpenStack\Log\* \\'$DEVSTACK_FLOATING_IP'\openstack\logs\hyperv02\'
```

Comment:
You are joking, right? I expect a function to be called twice, no copy-paste of 20-30 lines of code as duplicate!